ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__arm64-stable-4.12-upgrade-from-eus-4.10.yaml

```diff
@@ -0,0 +1,105 @@
+base_images:
+  ansible:
+    name: "4.10"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.10"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  openstack-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.12"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.10"
+  tools:
+    name: "4.10"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.10"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.11"
+  intermediate-arm64:
+    release:
+      architecture: arm64
+      channel: candidate
+      version: "4.11"
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.10"
+  latest-arm64:
+    release:
+      architecture: arm64
+      channel: candidate
+      version: "4.10"
+  target:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.12"
+  target-arm64:
+    release:
+      architecture: arm64
+      channel: candidate
+      version: "4.12"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: azure-ipi-p2-f14
```

Comment:
I think azure arm get supported from 4.12, but not 4.10.

Reply:
ah, thanks for pointing it out. Seems like only aws is supported in 4.10 arm64. Let me update it.